.gitmodules

```diff
@@ -2,9 +2,6 @@
 	path = external/cilium
 	url = https://github.com/saxena-anurag/cilium.git
 	branch = anurag/cilium_xdp_bpf_code_changes
-[submodule "external/ebpf-for-windows"]
```

Comment:
We should also update the Getting Started (`https://github.com/microsoft/ebpf-for-windows-demo/blob/main/docs/GettingStarted.md`)as ebpf-for-windows is not a submodule, though we can do this in a separate PR.